mgr/src/cmds/rebalance_process.cr

```diff
@@ -139,6 +139,7 @@ class Rebalancer
 
         backend_full_path = Path.new(@backend_dir, rel_path)
         if File.directory?(backend_full_path)
+          entries = Dir.new(Path.new(@mount_dir, rel_path).to_s).entries
```

Comment:
To supress lint error you can just use `Dir.new(Path.new(@mount_dir, rel_path).to_s).entries` or `_entries = Dir.new(Path.new(@mount_dir, rel_path).to_s).entries`

@amarts Doing `ls`  fixing the `ENOENT` issue. Do you think this is due to cache issue in mount or stat required after fix-layout?  Doing `ls` on mount for every directory may become expensive.

Reply:
Agree to last point. But if `ls` is required here compulsorily can skip setting of migrate-data xattrs for empty dirs too as a side-effect.

Reply:
Try one thing. Add `File.exists?` check before running setxattr. If it is stat issue then `File.exists?` will resolve it.